03_qgep_db_dss.sql

```diff
@@ -36,7 +36,7 @@ vsacode integer NOT NULL,
 value_en character varying(50),
 value_de character varying(50),
 value_fr character varying(50),
-value_it character varying(50),
+value_it character varying(60),
```

Comment:
I would propose to keep all languages the same length.
Possibly also just drop the maximum length for less troubles.